bl_ui_widgets/bl_ui_widget.py

```diff
@@ -97,7 +97,10 @@ def update(self, x, y):
             (self.x_screen + self.width, y_screen_flip),
         )
 
-        self.shader = gpu.shader.from_builtin("2D_UNIFORM_COLOR")
+        if bpy.app.version < (3, 4, 0):
```

Comment:
https://wiki.blender.org/wiki/Reference/Release_Notes/4.0/Python_API

GPU Module (gpu)

- Remove deprecated 2D_ / 3D_ prefix for built-in shader names ([9a8fd2f1dd](https://projects.blender.org/blender/blender/commit/9a8fd2f1ddb491892297315a4f76b6ed2b0c1b94)).